Asciidoctor extensions don't work on Windows (cannot find module)

Thanks for the effort. I'm trying to use an `Asciidoctor.js` extension (like `asciidoctor-emoji`), but it says the module cannot be found.

## Description

The version of `asciidoctor-vscode` is 3.0.1.

It might be because I use `nvm` for the workspace.

## System Information

```
Version: 1.71.0 (user setup)
Commit: 784b0177c56c607789f9638da7b6bf3230d47a8c
Date: 2022-09-01T07:36:10.600Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19043
Sandboxed: No
```

## To Reproduce

Steps to reproduce the issue:

1. Go to '...'
2. Click on '....'
3. Etc...

My settings are:

```json
{
    "asciidoc.preview.useEditorStyle": false,
    "asciidoc.preview.scrollEditorWithPreview": false,
    "asciidoc.preview.scrollPreviewWithEditor": false,
    "asciidoc.preview.refreshInterval": 0,
    "asciidoc.preview.doubleClickToSwitchToEditor": false,
    "asciidoc.extensions.registerWorkspaceExtensions": true,
}
```

The error message is:

```
/c:/Users/xxx/GitHub/docs/.asciidoctor/lib/emoji.js: Error: Cannot find module '/c:/Users/xxx/GitHub/docs/.asciidoctor/lib/emoji.js' Require stack: - c:\Users\xxx\.vscode\extensions\asciidoctor.asciidoctor-vscode-3.0.1\dist\src\asciidocParser.js - c:\Users\xxx\.vscode\extensions\asciidoctor.asciidoctor-vscode-3.0.1\dist\src\image-paste.js - c:\Users\xxx\.vscode\extensions\asciidoctor.asciidoctor-vscode-3.0.1\dist\src\commands\pasteImage.js - c:\Users\xxx\.vscode\extensions\asciidoctor.asciidoctor-vscode-3.0.1\dist\src\commands\index.js - c:\Users\xxx\.vscode\extensions\asciidoctor.asciidoctor-vscode-3.0.1\dist\src\extension.js - c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js - c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js - c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
```

--- a/src/asciidocParser.ts
+++ b/src/asciidocParser.ts
@@ -310,7 +310,7 @@ export class AsciidocParser {
     }
     const extfiles = await this.getExtensionFilesInWorkspace()
     for (const extfile of extfiles) {
-      const extPath = extfile.path
+      const extPath = extfile.fsPath
       try {
         delete require.cache[extPath]
         const extjs = require(extPath)

